src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -547,6 +547,17 @@ private bool ProcessAnyFileContent(IRequestInfo info, string localPath)
 				if (startOfBookEdit > 0)
 					path = BloomFileLocator.GetBrowserFile(localPath.Substring(startOfBookEdit));
 			}
+
+			if (localPath.StartsWith("pageChooser/") && IsImageTypeThatCanBeReturned(localPath))
+			{
+				var templatePath = Path.Combine(CurrentBookHandler.CurrentBook.FindTemplateBook().FolderPath,
```

Comment:
Would a correct comment for this kinda complicated line be

    // if we're in the page chooser dialog and looking for an thumbnail representing a 
    // template page, look that that thumbnail in the book that is the template source, 
    // rather than in the book we are editing.

?